Dockerfile

```
@@ -0,0 +1,16 @@
+FROM golang:1.22.3-alpine
+
+WORKDIR /app
+
+COPY go.mod ./
+COPY go.sum ./
```

no. of layers can be reduced. 
```suggestion
COPY go.mod go.sum ./
```